org.alloytools.kodkod.core/src/main/java/kodkod/engine/solveengine/Model.java

```diff
@@ -0,0 +1,108 @@
+package kodkod.engine.solveengine;
+
+import kodkod.engine.solveengine.enums.SEStatusCode;
+import kodkod.engine.solveengine.enums.SolveStatusCode;
+
+import java.io.UnsupportedEncodingException;
+import java.net.UnknownHostException;
+import java.rmi.ServerException;
+
+public abstract class Model {
+    /** The api-key used by solve engine to recognise the user */
+    protected String token;
+
+    /** The name of the problem */
+    protected String fileName;
+
+    /** Will print the steps of the process going, if true */
```

Comment:
for now yes, but anyway the model is always initialised with interactive = False, for now
Should I remove it ? Or keep it in case the possibility is added in the future ? (with then a defined printing output)